examples/step-63/step-63.cc

```diff
@@ -197,18 +197,16 @@ namespace Step63
       CompareDownstream<typename DoFHandler<dim>::level_cell_iterator, dim>
         comparator(direction);
 
-    typename DoFHandler<dim>::level_cell_iterator cell = dof.begin(level);
-    typename DoFHandler<dim>::level_cell_iterator endc = dof.end(level);
-    for (; cell != endc; ++cell)
+    for (const auto &cell : dof.cell_iterators_on_level(level))
```

Comment:
It was pre-existing, but would you mind changing `dof` to `dof_handler` to match all the other examples?